api/src/main/java/org/pmiops/workbench/featuredworkspace/FeaturedWorkspaceServiceImpl.java

```diff
@@ -49,23 +51,23 @@ public Optional<FeaturedWorkspaceCategory> getFeaturedCategory(DbWorkspace dbWor
 
   public List<WorkspaceResponse> getWorkspaceResponseByFeaturedCategory(
       FeaturedWorkspaceCategory featuredWorkspaceCategory) {
-    DbFeaturedCategory requestedDbCategory =
-        featuredWorkspaceMapper.toDbFeaturedCategory(featuredWorkspaceCategory);
 
-    return featuredWorkspaceDao.findDbFeaturedWorkspacesByCategory(requestedDbCategory).stream()
-        .map(
-            dbFeaturedCategory -> {
-              DbWorkspace dbWorkspace = dbFeaturedCategory.getWorkspace();
-              RawlsWorkspaceResponse rawlsWorkspaceResponse =
-                  fireCloudService.getWorkspace(
```

Comment:
I have not investigated why this call was returning 404s (TODO!) but avoiding it will mean that the API returns a list of workspaces - if possibly incomplete.